README.md

```diff
@@ -1,17 +1,28 @@
 
-TotalFreedomMod is a CraftBukkit server plugin designed primarily to support the [Official TotalFreedom Minecraft Server](https://totalfreedom.me/). However, you are more than welcome to adapt the source for your own server.
+TotalFreedomMod (TFM) is a Spigot server plugin, designed primarily to support the [TotalFreedom Minecraft Server](https://totalfreedom.me/). However, you are more than welcome to adapt the source for your own server.
 
-This plugin was originally coded by StevenLawson (Madgeek1450), with Jerom van der Sar (Prozza) becoming heavily involved in its development some time later. It consists of over 85 custom coded commands and a large variety of distinguishable features not included in any other plugin. The plugin has since its beginning grown immensely. Together, with the main TotalFreedom server, TotalFreedomMod has a long-standing reputation of effectiveness whilst maintaining a clear feeling of openness towards the administrators and the players themselves.
+TotalFreedomMod consists of over 175 custom-coded commands, and a huge variety of unique features. For the better part of a decade, TFM has been trusted to provide TF with the administration tools, player utilities, community integration, and fun surprises that define the TotalFreedom experience. TFM is always growing to fulfill the community's wants and needs.
 
-### Contributing
+Originally coded by Steven Lawson (Madgeek1450) and Jerom van der Sar (Prozza), TotalFreedomMod is a community effort with a dedicated and passionate developer base.
 
-Please see [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in developing TotalFreedomMod.
+## Contributing
+Please see [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in developing TotalFreedomMod. Pull requests welcome!
 
-For information on how TotalFreedomMod is licensed, please see [LICENSE.md](LICENSE.md).
+Please see [LICENSE.md](LICENSE.md) for information on how TotalFreedomMod is licensed.
 
-For information on our security policy and reporting an issue, please see [SECURITY.md](SECURITY.md)
+Please see [SECURITY.md](SECURITY.md) for information on our security policy and how to report an issue.
 
-### Compiling
+## Compiling
+1) Clone this repository. If you would rather not use Git on the command line, GitHub Desktop [(Windows/macOS)](https://desktop.github.com/) [(Linux)](https://github.com/shiftkey/desktop) makes contributing easy.
+2) You will need an IDE for developing plugins. We recommend [IntelliJ IDEA](https://www.jetbrains.com/idea/). This plugin is built using Maven, which is included with IDEA.
+3) Install the latest [OpenJDK 11 (LTS)](https://adoptopenjdk.net/) and configure your IDE to use it. In IDEA, this can be set under **File** < **Project Structure...** < **Project SDK**.
+
+> **Note:** As of the upcoming 1.18 update, TotalFreedomMod will require JDK 16 or newer going forward.
+
+4) Create a new Maven Configuration with the following: `clean install`
```

Comment:
Steps 4 and 5 shouldn't really be needed afaik. If you run an "mvn clean build" it should build the project and compile it as normal and give you an output in the "Target" directory of your project. Were you not seeing this?